### PR TITLE
PADV-1173 feat: add assigned classes section

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-1173

### Description
Add assigned classes section

### Changes made
Add assigned classes section
Add styles, colors and variables
Add test

### Screenshots
Screenshot from 2024-07-18 14-03-55

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1990/dashbaord

### CODE REVIEW

File: src/features/Dashboard/AssignedClasses/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const [classCards, setClassCards] = useState([]);
+  const numberOfClasses = 3;
+
+  useEffect(() => {
+    // Display only the first 'NumberOfClasses' on the homepage.
+    if (classes.length > numberOfClasses) {
+      setClassCards(classes.slice(0, numberOfClasses));
+    } else {
+      setClassCards(classes);
+    }
```
Comment (01001110J) [inline src/features/Dashboard/AssignedClasses/index.jsx:22]: What about:

```suggestion
    setClassCards(classes.length > numberOfClasses ? classes.slice(0, numberOfClasses) : classes);
```

File: src/features/Dashboard/AssignedClasses/index.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { Button } from 'react-paragon-topaz';
+
+import ClassCard from 'features/Dashboard/AssignedClasses/ClassCard';
+
+import 'features/Dashboard/AssignedClasses/index.scss';
+
+const AssignedClasses = () => {
+  const classes = useSelector((state) => state.common.allClasses.data);
+  const [classCards, setClassCards] = useState([]);
+  const numberOfClasses = 3;
```
Comment (01001110J) [inline src/features/Dashboard/AssignedClasses/index.jsx:14]: `numberOfClasses` the name is not very clear, what about something like: `visibleClassCount` or `displayClassLimit`.

File: src/features/Dashboard/AssignedClasses/ClassCard.jsx
[the remark this answers is not in the document]
Comment (AuraAlba) [reply src/features/Dashboard/AssignedClasses/ClassCard.jsx:1]: The title change is an error, the icon is not part of the MVP and for the second I think is also an error, make not sense have two lines

File: src/features/Dashboard/AssignedClasses/index.jsx (hunk 1 above)
Comment (AuraAlba) [reply src/features/Dashboard/AssignedClasses/index.jsx:22]: It is easier to read with the if and else

**Review by 01001110J — APPROVED** [review]

**Review by anfbermudezme — APPROVED** [review]

**Pull request merged**